proto/sysinfo.proto

```diff
@@ -479,3 +479,27 @@ message HardwareInfo {
     }];
 
 }
+
+// OSX Plist Start Calendar Interval
+
+message LaunchdStartCalendarInterval {
+  optional int32 minute = 1;
+  optional int32 hour = 2;
+  optional int32 day = 3;
+  optional int32 weekday = 4;
+  optional int32 month = 5;
+}
+
+// OSX Plist Files
+message LaunchdPlist {
+  optional string label = 1;
+  optional bool disabled = 2;
+  optional string username = 3;
```

Comment:
I am happy to change these all back to the normal case.  I only made them lower and added _ to fit with the rest of the definitions.  If making them match exactly isn't a problem I will just do that.